sarabajraktari/car2go
# Additional Fields for Cards Module, bug-fixes

This push contains the following bugfixes:

**Bugfix**:
1. Bugfix on contact-form.scss, modified spinner style caused an undesired rotating circle top left of the page
2. Fixed ACF for FAQ e-mail, corrected max-width which was visible on smaller screen-sizes.
3. Bugfix on load-more.js, added an if statement to check if the element is actually existent before applying the classes.

**Changed Made**:
Added 4 Fields to Cards and their ACF into car_details in order to specify the necessary details.

**How to test**:
1. Import ACF car_details.json
2. Go to Cars Select the Car you want to add the Rent Details, it should show at the top of Car Details just under the text editor.
3. Refresh Front Page, or Navigate to Cars page to see the results.
